core/app/models/spree/payment_method/check.rb

```diff
@@ -18,7 +18,9 @@ def capture(*args)
       simulated_successful_billing_response
     end
 
-    def cancel(response); end
+    def cancel(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `cancel(*)` if you want the method to accept any arguments but don't care about them.